Update init-theme.js

bschroeter opened the pull request:
Fixing a syntax error for the apiInitializer.

bschroeter committed:
Update init-theme.js

Fixing a syntax error for the apiInitializer.

Whyborn commented on javascripts/discourse/api-initializers/init-theme.js:
```diff
@@ -1,5 +1,5 @@
 import { apiInitializer } from "discourse/lib/api";
 
-export default apiInitializer((api)) => {
```
Should this be

```
export default apiInitializer((api) => {
```

as per the discourse meta thread [here](2f3ff3e80d6d174d93048c723ea6ad605798c841)? I don't know what the difference is

Whyborn commented on javascripts/discourse/api-initializers/init-theme.js:
```diff
@@ -1,5 +1,5 @@
 
-export default apiInitializer((api)) => {
-    api.replaceIcon('right-to-bracket', 'access-nri-aaf-icon-32x32')
+export default apiInitializer(api => {
```
Should this be

```
export default apiInitializer((api) => {
```

as per the discourse meta thread [here](https://meta.discourse.org/t/modernizing-inline-script-tags-for-templates-js-api/366482)? I don't know what the difference is

Whyborn commented:
I added a comment to the PR- can you see it?

Whyborn committed:
Update init-theme.js

Just to make it exactly equivalent to their recommendation- don't think it actually makes a difference
